<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>862</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCalc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="SmartCalc">
       <widget class="QLineEdit" name="CalcDisplay">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>3</y>
          <width>782</width>
          <height>291</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>0.0</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="CalculateButton">
        <property name="geometry">
         <rect>
          <x>702</x>
          <y>300</y>
          <width>80</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ExpressionInput">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>300</y>
          <width>696</width>
          <height>25</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string extracomment="Enter your expression here !">Enter your expression here !</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Credit">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Total credit amount:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="creditAmount">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>10</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>40</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Term:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="term">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>50</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="termBox">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>50</y>
          <width>86</width>
          <height>25</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>Years</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Months</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>90</y>
          <width>91</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Interest rate:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="interestRate">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>90</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QRadioButton" name="anButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>150</y>
          <width>112</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Annuity</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="diffButton">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>170</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Differentiated</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>120</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="creditCalcButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>220</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="creditOutput">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>10</y>
          <width>301</width>
          <height>261</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>847</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionSmartCalc"/>
    <addaction name="actionCredit"/>
   </widget>
   <addaction name="menumode"/>
  </widget>
  <action name="actionSmartCalc">
   <property name="text">
    <string>SmartCalc</string>
   </property>
  </action>
  <action name="actionCredit">
   <property name="text">
    <string>Credit</string>
   </property>
  </action>
  <action name="actionDeposit">
   <property name="text">
    <string>Deposit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
